client/my-sites/upgrades/cart/cart-plan-discount-ad.jsx

```diff
@@ -50,10 +51,35 @@ const CartPlanDiscountAd = React.createClass( {
 			return null;
 		}
 
+		if ( plan.isDomainUpgrade ) {
+			return (
+				<CartAd>
+					<p>{ translate(
+						"You're getting a %(discount)s discount off the regular price of the plan (%(originalPrice)s)" +
+						", because you already paid for the domain.",
+						{
+							args: {
+								discount: plan.formattedDiscount,
+								originalPrice: plan.formattedOriginalPrice
+							}
+						}
+					) }</p>
+					<p>{ translate(
```

Comment:
It looks bad if you use `<p>` here. Should probably follow the same pattern as below. :)

![upsell message](https://cloud.githubusercontent.com/assets/1103398/23659796/9bc230b6-0347-11e7-80a6-1348c07fff07.png)